src/components/backup/useCreateBackup.ts

```diff
@@ -53,10 +53,11 @@ export const useCreateBackup = () => {
 
   const onSuccess = useCallback(
     async (password: string) => {
-      const hasSavedPassword = await getLocalBackupPassword();
-      if (!hasSavedPassword && password.trim()) {
+      if (backupsStore.getState().storedPassword !== password) {
```

Comment:
this will prompt a faceID if the storedPassword is either a) not set, or b) different than the backup password